### PR TITLE
Small refactor to include CommandExists method in Runner interface

### PR DESCRIPTION
**Pull request by davidcassany**
Signed-off-by: David Cassany <dcassany@suse.com>

### CODE REVIEW

File: pkg/types/v1/runner.go [hunk 1]
```diff
@@ -42,10 +49,13 @@ func (r RealRunner) RunCmd(cmd *exec.Cmd) ([]byte, error) {
...
-	if r.Logger != nil {
-		r.Logger.Debugf("Running cmd: '%s %s'", command, strings.Join(args, " "))
+	if !r.CommandExists(command) {
+		err := fmt.Errorf("Command %s not found", command)
+		r.error(err.Error())
+		return []byte{}, err
 	}
+	r.debug(fmt.Sprintf("Running cmd: '%s %s'", command, strings.Join(args, " ")))
+	cmd := r.InitCmd(command, args...)
 	return r.RunCmd(cmd)
```
Comment (frelon) [inline pkg/types/v1/runner.go]: RunCmd will return an ErrNotFound if the command is not found, so the new test passes with this change instead:

```
        bytes, err := r.RunCmd(cmd)
        if err != nil {
            r.Logger.Errorf("Error running command: %s", err.Error())
        }
        
	return bytes, err
```

**Review by frelon — COMMENTED** [review]
I don't think it's needed to use the CommandExists method in RealRunner Run-method, otherwise looks fine :+1:

File: pkg/types/v1/runner.go (hunk 1 above)
Comment (davidcassany) [reply pkg/types/v1/runner.go]: Right, you got a point, gonna adapt it

**Review by frelon — APPROVED** [review]
LGTM

**Pull request merged**